### PR TITLE
initial setup nextjs

### PR DESCRIPTION
**Pull request by akshaynr**
Implemented server side rendering using NextJS

Screenshot 2024-05-02 at 11 42 39 AM
Screenshot 2024-05-02 at 11 42 53 AM

### CODE REVIEW

File: netflix-roulette/app/page.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import Image from "next/image";
+import styles from "./page.module.css";
+
+export default function Home() {
+  return (
+    <main className={styles.main}>
+      <div className={styles.description}>
+        <p>
+          Get started by editing&nbsp;
```
Comment (shubhamagarwal) [inline netflix-roulette/app/page.js]: why do we need &nbsp here ?

**Comment by shubhamagarwal** [comment]
@akshaynr  can you add screenshot for this PR ?

File: netflix-roulette/app/page.js (hunk 1 above)
Comment (akshaynr) [reply netflix-roulette/app/page.js]: removed it

**Comment by akshaynr** [comment]
> @akshaynr can you add screenshot for this PR ?

Added screenshot

**Pull request merged**